README.md

```diff
@@ -6,295 +6,354 @@
 
-To configure a management namespace named `safe`, there must be a configmap named `ibm-cloud-operator` created in the same namespace as the IBM Cloud Operator itself. This configmap indicates the name of the management namespace, in a field `namespace`. To create such a config map, execute the following:
+When you create an IBM Cloud Operator service or binding resource, the operator checks the namespace that you create the resource in for the secret and configmap. If the the operator does not find the secret and configmap, it checks its own namespace for a configmap that points to a [management namespace](#setting-up-a-management-namespace). Then, the operator checks the management namespace for `<namespace>-secret-ibm-cloud-operator` secrets and `<namespace>-config-ibm-cloud-operator` configmaps. If no management namespace exists, the operator checks the `default` namespace for the `secret-ibm-cloud-operator` secret and `config-ibm-cloud-operator` configmap.
 
-```bash
-cat <<EOF | kubectl apply -f -
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: ibm-cloud-operator
-  namespace: <namespace where IBM Cloud Operator has been installed>
-  labels:
-    app.kubernetes.io/name: ibmcloud-operator
-data:
-  namespace: safe
-EOF
-```
+You can override the account context in the `Service` configuration file with the `context` field, as described in the following table. You might override the account context if you want to use a different IBM Cloud account to provision a service, but do not want to create separate secrets and configmaps for different namespaces.
 
-This configmap indicates to the operator where to find the management namespace, in this case `safe`.
-Next the `safe` namespace needs to contain secrets and configmaps corresponding to each namespace that will contain services and bindings. The naming convention is as follows:
+| Field            | Required | Type     | Description |
+|:-----------------|:---------|:---------|:------------|
+| org              | No       | `string` | The Cloud Foundry org. To list orgs, run `ibmcloud account orgs`. |
+| space            | No       | `string` | The Cloud Foundry space. To list spaces, run `ibmcloud account spaces`. |
+| region           | No       | `string` | The IBM Cloud region. To list regions, run `ibmcloud regions`. |
+| resourceGroup    | No       | `string` | The IBM Cloud resource group name. You must also include the `resourceGroupID`. To list resource groups, run `ibmcloud resource groups`. |
+| resourceGroupID  | No       | `string` | The IBM Cloud resource group ID. You must also include the `resourceGroup`. To list resource groups, run `ibmcloud resource groups`. |
+| resourceLocation | No       | `string` | The location of the resource.<!--what is this?--> |
```

Comment:
> `resourceLocation`: The location of the resource.

What is this? like the data center the resource is created in, within the region? how would the user know where the location is?

Reply:
I.. don't remember 😅 I might have to go digging to find out why this is here, when we also have `region`